packages/@sfx/core/test/unit/common/index.test.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from '../../utils';
+import {
+  Core as CoreExport,
+  Plugin as PluginExport,
+  PluginMetadata as PluginMetadataExport,
+  PluginRegistry as PluginRegistryExport,
+} from '../../../src';
+import Core from '../../../src/core';
+import { Plugin, PluginMetadata, PluginRegistry } from '../../../src/plugin';
+import * as CoreUtils from '../../../src/utils/core';
```

Comment:
`CoreUtils` is not being used in this file.

Reply:
He's not the import referee we deserved. He's the import referee we needed.